build/win32/CMakeLists.txt

```diff
@@ -2,13 +2,13 @@ cmake_minimum_required(VERSION 3.24)
 
 set(BASE_DIR ${CMAKE_CURRENT_LIST_DIR}/../..)
 
-option(WITHOUT_LMDB    "Include LMDB support"    OFF)
-option(WITHOUT_LUA     "Include LUA support"     OFF)
-option(WITHOUT_LIBXML2 "Include LibXML2 support" OFF)
-option(WITHOUT_MAXMIND "Include MaxMind support" OFF)
-option(WITHOUT_CURL    "Include CURL support"    OFF)
+option(WITH_LMDB    "Include LMDB support"    OFF)
```

Comment:
I transformed the options from WITHOUT_XXX to WITH_XXX to make them easier to use (and not have to think about double negatives to include a feature or not).

At the same time, while working on this PR I noticed that the default Windows build was using LMDB instead of the in-memory backend for collections, which is the default for non-Windows platforms. So I adjusted this to align them (and thus changed the GitHub workflow too, to align the names of the configurations).